examples/time_series_forecasting.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<ul><li>replace "/resampling" with "or resampling"? </li><li>I suggest putting the "Note" on a new line</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/giotto-ai/giotto-tda/pull/475/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
"or" doesn't quite cut it, and "and" would maybe be confusing with "both". Any alternative? Done about the "Note".

Reply:
<p>ah yes you're right. how about </p><p> </p><p>"which combines both the sliding window transformation on <code>X</code>  and resampling of <code>y</code>  with the feature extraction from the windows on <code>X</code> "?</p>

Reply:
Done,